### PR TITLE
x11-misc/mugshot: New package

### PR DESCRIPTION
**Pull request by ConiKost**
x11-misc/mugshot: New package

Closes: https://bugs.gentoo.org/571382

### CODE REVIEW

File: x11-misc/mugshot/files/fix_env_spawn_args.patch [hunk 1]
```diff
@@ -0,0 +1,47 @@
+--- a/mugshot_lib/SudoDialog.py	2018-04-12 00:23:21.000000000 +0200
```
Comment (mgorny) [inline x11-misc/mugshot/files/fix_env_spawn_args.patch]: Are those patches accepted upstream? Include a link to their source.

File: x11-misc/mugshot/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
```
Comment (mgorny) [inline x11-misc/mugshot/metadata.xml]: The usual stuff.

File: x11-misc/mugshot/mugshot-0.4.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eapi7-ver gnome2-utils
+
+MY_PV="$(ver_cut 1-2)"
```
Comment (mgorny) [inline x11-misc/mugshot/mugshot-0.4.0.ebuild]: Used only once.

File: x11-misc/mugshot/mugshot-0.4.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+DESCRIPTION="A lightweight user-configuration application"
+HOMEPAGE="https://launchpad.net/mugshot"
+SRC_URI="https://launchpad.net/${PN}/${MY_PV}/${PV}/+download/${PN}-${PV}.tar.gz"
```
Comment (mgorny) [inline x11-misc/mugshot/mugshot-0.4.0.ebuild]: ${PN}-${PV} is equal to…

File: x11-misc/mugshot/mugshot-0.4.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+LICENSE="GPL-3"
```
Comment (mgorny) [inline x11-misc/mugshot/mugshot-0.4.0.ebuild]: Did you verify it?

File: x11-misc/mugshot/mugshot-0.4.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gnome libreoffice pidgin webcam"
+
+RDEPEND="dev-python/dbus-python[${PYTHON_USEDEP}]
+	dev-python/pexpect[${PYTHON_USEDEP}]
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	sys-apps/accountsservice
+	x11-libs/gtk+:3
```
Comment (mgorny) [inline x11-misc/mugshot/mugshot-0.4.0.ebuild]: Don't you need introspection?

File: x11-misc/mugshot/files/fix_env_spawn_args.patch (hunk 1 above)
Comment (ConiKost) [reply x11-misc/mugshot/files/fix_env_spawn_args.patch]: The `fix_env_spawn_args.patch` is written by myself. Without this one, Mugshot seems not to work on Gentoo, it doen's even start then. There is a ticket: https://bugs.launchpad.net/ubuntu/+source/mugshot/+bug/1443283. I've submited the patch there.

The two other patches are from Arch Linux. But it seems, upstream does not care about them.

File: x11-misc/mugshot/metadata.xml (hunk 2 above)
Comment (ConiKost) [reply x11-misc/mugshot/metadata.xml]: Okay, made a better one.

File: x11-misc/mugshot/mugshot-0.4.0.ebuild (hunk 3 above)
Comment (ConiKost) [reply x11-misc/mugshot/mugshot-0.4.0.ebuild]: So added directly to SRC_URI, if you mean that?

File: x11-misc/mugshot/mugshot-0.4.0.ebuild (hunk 4 above)
Comment (ConiKost) [reply x11-misc/mugshot/mugshot-0.4.0.ebuild]: Thanks, fixed.

File: x11-misc/mugshot/mugshot-0.4.0.ebuild (hunk 5 above)
Comment (ConiKost) [reply x11-misc/mugshot/mugshot-0.4.0.ebuild]: Fixed.. Why nearly all sites write "GPL-X" but not "GPL-X+" -.-

File: x11-misc/mugshot/mugshot-0.4.0.ebuild (hunk 6 above)
Comment (ConiKost) [reply x11-misc/mugshot/mugshot-0.4.0.ebuild]: Seems so, when I remove it, mugshot won't start and complains. Added.

File: x11-misc/mugshot/mugshot-0.4.0.ebuild (hunk 3 above)
Comment (mgorny) [reply x11-misc/mugshot/mugshot-0.4.0.ebuild]: Yes, that exactly.

File: x11-misc/mugshot/mugshot-0.4.0.ebuild (hunk 5 above)
Comment (mgorny) [reply x11-misc/mugshot/mugshot-0.4.0.ebuild]: If by 'sites' you mean GitHub, then GitHub just verifies whatever file is placed as 'COPYING' or 'LICENSE'. It doesn't check exact copyright notices in source files.

**Pull request closed without merging**